recipes/gmm/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# from https://web.archive.org/web/20200225211624/http://getfem.org/gmm/install.html
+# configure first using conda prefix
+./configure --prefix=${PREFIX}
+# make it
+make -j$CPU_COUNT
```

Comment:
Is there a `make check` we can run here?

Reply:
looks like there is a `make check` and also a `make installcheck`, I didn't know if there was a check target to use, anyways I'll add the `make check`